esp/logging/loggingmanager/loggingmanager.cpp

```diff
@@ -161,10 +161,12 @@ bool CLoggingManager::updateLog(IEspContext* espContext, const char* option, IPr
             espContextTree.setown(createPTree("ESPContext"));
 
             short port;
-            StringBuffer sourceIP;
+            StringBuffer sourceIP, peerStr;
             const char* esdlBindingID = espContext->queryESDLBindingID();
             espContext->getServAddress(sourceIP, port);
             espContextTree->addProp("SourceIP", sourceIP.str());
+            espContext->getPeer(peerStr);
+            espContextTree->addProp("EndUserIP", peerStr.str());
```

Comment:
Would it be better for the property to be `Peer`? It would keep the internal and external names in sync, which could reduce confusion in people familiar with both. It might also increase confusion in people only dealing with the configuration.

Reply:
Ok, I did.